air/src/options.rs

```diff
@@ -246,6 +252,19 @@ impl ProofOptions {
     pub fn partition_options(&self) -> PartitionOptions {
         self.partition_options
     }
+
+    /// Returns the `[BatchingType]` defining the method used for batching the multi-point quotients
+    /// defining the DEEP polynomial.
+    ///
+    /// Linear batching implies that independently drawn random values per multi-point quotient
+    /// will be used to do the batching, while Algebraic batching implies that powers of a single
+    /// random value are used.
+    ///
+    /// Depending on other parameters, Algebraic batching may lead to a small reduction in the security
+    /// level of the generated proofs, but avoids extra calls to the random oracle (i.e., hash function).
+    pub fn batching_used_for_deep_poly(&self) -> BatchingType {
```

Comment:
nit: let's rename this to `deep_poly_batching_method()` and also, let's rename `BatchingType` to `BatchingMethod`.

Reply:
changed